src/Services/QueuedJobService.php

```diff
@@ -1533,6 +1579,271 @@ private function addJobHandlersToLogger(LoggerInterface $logger, QueuedJob $job,
         }
     }
 
+    /**
+     * Find any eligible jobs that qualify for an automated job retry and schedule their retry
+     *
+     * @throws ValidationException
+     */
+    protected function retryEligibleJobs(int $queueType): void
```

Comment:
This method has gotten very long - are there pieces of this that could be sensibly extracted into `private` methods to make it easier to skim read and maintain?

e.g. the loop over `$jobRetryStatusMap` that pull out empty or invalid statuses might make sense to extract (and resolves the variable naming problem as a bonus)